views/userManagement.njk

```diff
@@ -54,12 +54,22 @@
     {% for item in obj %}
         <tr>
             <td>{{item.first}} {{item.last}}</td>
-            <td>{{item.badge}}</td>
+            <td><name="badgeNumber">{{item.badge}}</td>
             <td>{{item.confirmation}}</td>
-            <td><a href="#">User</a> <a href="#">Station</a> </td> 
+            <td><button class="btn btn-primary" onclick="fwd(this)" data-badge={{item.badge}} >User-Station</button></td>
+            {# <td action="/userManagement/badge" method="post"><button class="btn btn-primary" type="Submit">User-Station</button><td> #}
         </tr>
     {% endfor %}   
 </table>
 
+<script type="text/javascript">
+
+        function fwd(obj) {
+            var badge  = obj.getAttribute("data-badge")
+            window.location.replace("/userManagement/" + badge)
```

Comment:
When I click to see a user and then I am done and I click the back button, it does not go back to the user list. It may have to do with this.